.gitignore

```diff
@@ -44,3 +44,10 @@ yarn-debug.log*
 database.yml
 hyacinth.yml
 resque.yml
+solr.yml
```

Comment:
Can we remove solr.yml from the gitignore if it's not being used?

Reply:
Can we wait to make that change until after Hyc3 is out? I actually have to switch back and forth between the 2.x and 3.x branches a lot right now.

Reply:
Sure, but on that note I'd like to **highly** recommend checking out two copies of the repository, in different directories called hyacinth2 and hyacinth3.  I've had to deal with the same challenge and it's much easier that way.